File: OBI - Aula/quermesse.c
#include <stdio.h>

int main(void){
    int h,i,teste=0;
    scanf("%d", &h);
    while(h!=0){
        teste++;
        int vet[h+5];

        if(h==0)    break;
        
        for(i=0;i<h;i++){
            scanf("%d", &vet[i]);
        }

        for(i=0;i<h;i++){
            if(vet[i]==i+1){
                printf("Teste %d\n", teste);
                printf("%d\n\n", i+1);
            }
        }
    scanf("%d", &h);
    }
}

File: OBI - Aula/quermesse.py
teste=0

while(True):

    h = int(input())
    if h==0:
        break
    
    vet = [int(x) for x in input().split()]
    teste+=1

    for i in range(h):
        if(vet[i]==i+1):
            print("Teste", teste)
            print(i+1)
            print("")
            break
    
    